src/engraving/rendering/iscorerender.h

```diff
@@ -85,12 +85,12 @@ class TimeSig;
 }
 
 namespace mu::engraving::rendering {
-class IScoreRendering : MODULE_EXPORT_INTERFACE
+class IScoreRender : MODULE_EXPORT_INTERFACE
```

Comment:
To me, it would seem more natural to call it `IScoreRenderer`. The verb is "render", so the corresponding noun is "renderer".